### DISCUSSION TITLE
Option for sequential/parallel read

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#7820] [Support]
**From v1.2.3 release notes: Added an option to revert to sequential read. There are some systems (primarily macos+exfat) where the parallel read features results in very long lookup times. This addition makes the parallel feature the default, but adds the ability to disable it and revert back to sequential reads.**

Which option is this and how do we change it?  Could we selectively do sequential read on certain plot_directories while parallel reads on all others, on the same harvester?

**Comment by jayhohoho2019** [comment]
@wjblanke

**Comment by emlowe** [comment]
Edit config.yaml
```
harvester:
  parallel_read: False
```
No, you cannot mix and match

**Reply by jayhohoho2019** [reply]
Thanks.  I tried this on the harvester with a few exfat HDDs, and still see the same warning messages at similar frequencies (WARNING  Looking up qualities on ... took: (sometimes 30+ seconds).
